qemu/tests/blockdev_stream_backing_mask_on.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+from virttest import data_dir
+
+from provider import backup_utils
+from provider.blockdev_stream_base import BlockDevStreamTest
+from provider.virt_storage.storage_admin import sp_admin
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+class BlockdevStreamBackingMaskOnTest(BlockDevStreamTest):
+    """Do block-stream based on an existed snapshot in snapshot chain"""
+
+    def __init__(self, test, params, env):
+        super(BlockdevStreamBackingMaskOnTest, self).__init__(test, params, env)
+        self._snapshot_images = self.params.objects("snapshot_images")
+        backing_mask_protocol = self.params["backing_mask_protocol"]
+        if backing_mask_protocol:
+            self._stream_options["backing-mask-protocol"] = backing_mask_protocol
```

Comment:
self._stream_options["backing-mask-protocol"] = self.params["backing_mask_protocol"]

As we always set backing_mask_protocol in cfg, no need to do the check

Reply:
Please also check https://github.com/autotest/tp-qemu/pull/4141